releasenotes/notes/django-http-route-fix-913850f0d53a74b7.yaml

```
@@ -0,0 +1,3 @@
+fixes:
+  - |
+    django: Fixes an issue where 'http.route' was only set if use_handler_resource_format and use_legacy_resource_format were False.
```

minor tweak to the rst formatting

```suggestion
    django: Fixes an issue where ``http.route`` was only set if ``use_handler_resource_format`` and ``use_legacy_resource_format`` were set to ``False``.
```